_includes/configuration/images-file-types.md

```diff
@@ -2,7 +2,9 @@
 
 {{site.requires_5_6v}}
 
-This option configures which image file formats will be recognized and placed in an `img` element by the [`smart_paste`](#smart_paste) functionality when content is pasted into the editor.
+This option configures which image file formats are accepted by the editor. Changing this option will adjust the following editor behaviour:
+  - which image file formats are allowed to be uploaded in the [Image]({{site.baseurl}}/plugins/opensource/image/) dialog.
```

Comment:
Capital `W` at the start of the list items pls

Reply:
Fixed